# Request
string waypoints_file
---
# Response
bool success
string message